internal/driver/driver_focus.go

```diff
@@ -77,19 +77,44 @@ func compileTagFilter(name, value string, ui plugin.UI, err error) (func(*profil
 	if value == "" || err != nil {
 		return nil, err
 	}
+
+	tagValuePair := strings.SplitN(value, "=", 2)
+	var key string
```

Comment:
Maybe rename this to something like `wantKey` because below `key` is also used for other variables which is a bit confusing.